source/ODApplication.cpp

```diff
@@ -49,6 +51,11 @@
 void ODApplication::startGame(boost::program_options::variables_map& options)
 {
     ResourceManager resMgr(options);
+
+    LogManager logMgr;
+    logMgr.addSink(std::shared_ptr<LogSink>(new LogSinkConsole()));
```

Comment:
Yes, you have to use std::move to, well, move a unique ptr from a container to another. But it's feasible. Care to try?